### PR TITLE
changes

### PR DESCRIPTION
**Pull request by c-four77**
## Description

- Modify the target and the PolyZone to ox ones

- fixed a small typo in lib.hideTextUI()

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [ ] My code fits the style guidelines.
- [ ] My PR fits the contribution guidelines.

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -41,14 +40,14 @@ exports('SetVehicleStatus', setVehicleStatus)
 
 
 -- Functions
-
+
```
Comment (Manason) [inline client/main.lua:43]:
```suggestion

```

File: fxmanifest.lua [hunk 2]
```diff
@@ -3,7 +3,7 @@ game 'gta5'
 
 description 'QBX_MechanicJob'
 repository 'https://github.com/Qbox-project/qbx_mechanicjob'
-version '1.0.0'
+version '1.0.1'
```
Comment (Manason) [inline fxmanifest.lua:6]: Don't increment version on unreleased resource.

File: config.lua [hunk 3]
```diff
@@ -1,8 +1,8 @@
 Config = {}
 
 -- **** IMPORTANT ****
--- UseTarget should only be set to true when using qb-target
-Config.UseTarget = GetConvar('UseTarget', 'false') == 'true'
+Config.UseTarget = true
```
Comment (Manason) [inline config.lua:4]: It was defaulting to false before, now it defaults to true. Why the change?

File: client/main.lua [hunk 4]
```diff
@@ -70,168 +70,155 @@ local function registerDutyTarget()
...
-    plateZone:onPlayerInOut(function (isPointInside)
-        if isPointInside and QBX.PlayerData.job.onduty then
-            if plate.AttachedVehicle then
+    local plateZone = lib.zones.box({
+        coords = coords,
+        size = boxData.size,
+        rotation = boxData.rotation,
+        debug = Config.DebugPoly,
+        onEnter = function()
+            if QBX.PlayerData.job.onduty then
```
Comment (Manason) [inline client/main.lua:203]: good opportunity here for a guard clause to gate the unduly check https://manason.github.io/effective-fivem-lua/functions/#avoid-nesting

File: client/main.lua [hunk 5]
```diff
@@ -70,168 +70,155 @@ local function registerDutyTarget()
...
-                })
+    local size = Config.LocationsSize.vehicle
+
+    lib.zones.box({
+        coords = coords,
+        size = size,
+        rotation = 339.75,
+        debug = Config.DebugPoly,
+        onEnter = function()
+            if QBX.PlayerData.job.onduty then
```
Comment (Manason) [inline client/main.lua:168]: Good opportunity for a guard clause; https://manason.github.io/effective-fivem-lua/functions/#avoid-nesting

**Review by Manason — CHANGES_REQUESTED** [review]

File: client/main.lua (hunk 5 above)
Comment (c-four77) [reply client/main.lua]: I did changes with the guide u provided can u check it

File: config.lua (hunk 3 above)
Comment (c-four77) [reply config.lua]: so do i need to set it false or add the Convar back ?

Comment (Manason) [reply config.lua]: I think this is fine actually. It's a change to use target by default.
